docs/api/1.x/index.rst

```diff
@@ -3,6 +3,30 @@
 1.x
 ###
 
+Full reference
+==============
+
+Full detailed API documentation:
+
+.. toctree::
+   :maxdepth: 2
+
+   cliquet/utilities
+   authentication
+   cliquet/batch
+   cliquet/timestamps
+   cliquet/backoff
+   cliquet/errors
+   cliquet/deprecation
+   buckets
+   collections
+   records
+   groups
+   permissions
+   synchronisation
+   cliquet/resource
```

Comment:
Note: I wonder if we should not reorder them (ex. errors and deprecation after main objects buckets, collections...)